prepare/metrics/llm_as_judge_response_assesment.py

```diff
@@ -0,0 +1,14 @@
+from unitxt import add_to_catalog
+from unitxt.inference import PipelineBasedInferenceEngine
+from unitxt.llm_as_judge import LLMAsJudge
+
+
+inference_model = PipelineBasedInferenceEngine(model_name='google/flan-t5-large', max_new_tokens=32)
+
+metric = LLMAsJudge(inference_model=inference_model)
+
+add_to_catalog(
+    metric,
+    "metrics.llm_as_judge.model_response_assessment.mt_bench_flan_t5",
```

Comment:
metrics.rag.llm_as_judge.model_response_assessment.mt_bench_flan_t5

Reply:
I think may call it

metrics.rag.model_response_assessment.llm_as_judge_by_flan_t5_with_mt_bench_template
                ^               ^                                            ^                                         
               domain    metric goal                       implementation
               
similar to the existing:

metrics.rag.correctness.llama_index_by_gpt_3_5_turbo